packages/surveys/src/components/MultipleChoiceMultiQuestion.tsx

```diff
@@ -29,8 +29,15 @@ export default function MultipleChoiceSingleQuestion({
   brandColor,
 }: MultipleChoiceSingleProps) {
   const [otherSelected, setOtherSelected] = useState(
-    !!value && !question.choices.find((c) => c.label === value)
-  ); // initially set to true if value is not in choices
+    !!value &&
+      (value as string[]).some((item) => {
+        const choicesWithoutOther = question.choices
+          .filter((choice) => choice.id !== "other")
+          .map((item) => item.label);
+        return choicesWithoutOther.includes(item) === false;
+      })
```

Comment:
Can we write this in a cleaner way instead that is more readable? The functionality is working to the point, maybe we can clean this up a bit, since we currently do a:
```
[].some ->
    [].filter ->
        .map ->
            return
```

Maybe we optimise this too as well?